database/migrations/2023_08_16_184438_add_fee_to_transactions_table.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class () extends Migration {
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::table('transactions', function (Blueprint $table) {
```

Comment:
You'll need to update the schema dump in `database\schema\mysql-schema.sql` so unit tests will not fail.
You can do this by running `php vendor/bin/testbench schema:dump` and copy the dump from  `vendor/orchestra/testbench-core/laravel/database/schema`

Reply:
Nice